README.md

```diff
@@ -1,3 +1,11 @@
+# This repository has moved!
+
+It now lives in the repository of the [SonarJS analyzer](https://github.com/SonarSource/SonarJS/blob/master/packages/jsts/src/rules/README.md).
+
+eslint-plugin-sonarjs >2.0.0 makes all SonarJS rules available for ESLint users, instead of a subset as it was with ^1.0.0 living here.
+
```

Comment:
We could also mention that the plugin's name will not change.
Users will only need to upgrade the dependency.

Reply:
I would say that already pretty explicit with this line 
```
eslint-plugin-sonarjs >2.0.0 makes all SonarJS rules available for ESLint users, instead of a subset as it was with ^1.0.0 living here.
```

Reply:
👍